### PR TITLE
Issue #85 Circular Loading Component

### PR DESCRIPTION
**Pull request by Bus42**
# PR Details
Implements Material-UI Circular Loading Component

## Description

File changes to 
+ src/components/Loading/index.jsx

## Related Issue 
Issue #85 
Add Circular Progress Component ([PR_ID](https://github.com/SoftboxLab/juggernaut/issues/85))

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Adds requested component. Icon positioned absolutely, centered in viewport, and scaled (2x) for better visibility. Console 'signature' added (delete if not wanted).

### CODE REVIEW

File: src/components/Loading/index.jsx [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+
+const name = 'Loading Component'
+
+// sizes wrapper to viewport dimensions
+const wrapperStyle = {
+  width: '100vw',
+  height: '100vh'
+}
+
+const Loading = () => {
```
Comment (marlonmleite) [inline src/components/Loading/index.jsx]: This component does not follow our standards where it should use styled components and material-ui.

```jsx
import React from 'react'
import { CircularProgress } from '@material-ui/core';
import LoadingStyled from './styled'

function Loading () {
  return (
    <LoadingStyled>
      <CircularProgress />
    </LoadingStyled>
  )
}

export default Loading

```

File: src/components/Loading/index.jsx [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
-const Loading = () => <LoadingStyled>Loading...</LoadingStyled>
+const useStyles = makeStyles(theme => ({
+  progress: {
+    margin: theme.spacing(2)
+  }
+}))
+
+// positions loader in the center of the containing div element
+// enlarges animation for better visibility
+const loaderStyle = {
```
Comment (marlonmleite) [inline src/components/Loading/index.jsx]: Styles should be written in 1 file named style.js as per our standard:

**style.js**
```
const LoadingStyled = styled.div`
  width: 100%;
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
`

export default LoadingStyled
```

**Comment by Bus42** [comment]
Okay, I think I got it now and I think I created a new PR. Bear with me, I'm fairly new to contributing.

**Comment by Bus42** [comment]
Okay, I'm really confused now. Did you just ask me to rename style.js to style.js, or is that an older comment? There is no date or time stamp on the change request and I can't tell if the request was made before or after I pushed new changes yesterday (10//7). I kind of feel like I'm talking to a bot. Am I?

File: src/components/Loading/style.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 import styled from 'styled-components'
 
 const LoadingStyled = styled.div`
-  color: #777;
+width: 100vw;
```
Comment (vilaboim) [inline src/components/Loading/style.js]:
```suggestion
width: 100%;
```

File: src/components/Loading/style.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
...
+height: 100vh;
```
Comment (vilaboim) [inline src/components/Loading/style.js]:
```suggestion
height: 100%;
```

File: src/components/Loading/style.js [hunk 5]
```diff
@@ -1,7 +1,11 @@
 import styled from 'styled-components'
 
 const LoadingStyled = styled.div`
```
Comment (vilaboim) [inline src/components/Loading/style.js]: Fix properties indentation.

**Pull request merged**